app/src/main/java/com/github/epfl/meili/models/VisitedPointOfInterest.kt

```diff
@@ -0,0 +1,9 @@
+package com.github.epfl.meili.models
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import java.util.*
+
+@Parcelize
+data class VisitedPointOfInterest(var poi: PointOfInterest? = null, var dateVisited: Date? = Calendar.getInstance().time) :
```

Comment:
@tberkane I am unsure but do you think it would be best to change the name of the class to FavouritePointOfInterest? Maybe not, just wondering.